### ISSUE TITLE
indirect-buffer behavior not working [?]

### ISSUE THREAD

**Issue by ghost** [mwfogleman/org-randomnote#2]
Love the idea, thank for creating this mode!

I'm not sure if `indirect-buffer` is working as intended... it seems to pick a random heading in the org file setup as candidate (and switch to that buffer), but I think it's a regular full-view buffer... Is it suppose to create a new (indirect) buffer with a narrowed view of a particular header?

I've got this setup:

```emacs-lisp
  (use-package org-randomnote
    :init (spacemacs/set-leader-keys "aor" 'org-randomnote)
    :bind ("C-c r" . org-randomnote)
    :config (setq org-randomnote-candidates '("~/org/notes.org")
                  org-randomnote-open-behavior 'indirect-buffer))
```

**Comment by tasshin** [comment]
Sure, happy you're using it!

So, I don't really use indirect-buffers myself, so I'm not 100% sure if the way I set it up is the way that you'd want it done as someone who uses them.

When I set the open-behavior to indirect-buffer, it does create a new indirect buffer, at least as I understand how it's supposed to work. The buffer edits the original, but just shows the relevant header.

Let me know what you think. If it's not the way you expect, I'm open to PR's :)

**Comment by ghost** [comment]
Yes, that's what I thought, but for me, when selecting a random note, with the above setup, I still see all of the content (with cursor positioned on a random headline indeed).

**Comment by tasshin** [comment]
![screen shot 2017-12-15 at 11 39 18 am](https://user-images.githubusercontent.com/2863138/34051447-a67e246e-e18c-11e7-913f-a7ed526489a3.png)

This is what I get. Can you share a screenshot of what it looks like for you? Can you check that the variable is actually set to indirect-buffer?
